p2p/libp2p/libp2pContext_test.go

```diff
@@ -11,7 +11,10 @@ import (
 )
 
 func TestNewLibp2pContext_NilContextShoulsErr(t *testing.T) {
-	lctx, err := libp2p.NewLibp2pContext(nil, &mock.ConnectableHostStub{})
+	ctx := context.Background()
```

Comment:
please replace the three lines with declaration

Reply:
done.